test/cypress/integration/group-proposals.spec.js

```diff
@@ -167,6 +171,8 @@ describe('Proposals - Add members', () => {
   it('user1 see their accepted proposals to invite user4 and user6', () => {
     cy.giSwitchUser(`user1-${userId}`)
 
+    cy.giAddRandomIncome()
```

Comment:
Why do we need to add an income on these tests? It's about proposals, not income details 🤔

Reply:
It's because you can't see the proposal until you add your income detail. See @mmbotelho comment about the only widget available

Reply:
I don't think that makes sense. A member should see proposals even without income details. Imagine there's a proposal to change the mincome, and only after that sense, it makes sense for the user to add their income details.

With this new flow, they couldn't vote for it unless they add a mincome even if incorrect.

Reply:
The idea here is to guide the user through all the steps that they need to take, one by one. If you don't have your income details filled out, the app is useless to you, hence the emphasis on getting that done first before more "advanced", secondary mechanics are introduced. 

The example I used at the time was Portal (the game) and how the mechanics are slowly introduced to players until they feel natural and second nature.

Reply:
Hum... okay then, I trust the theory :)